typescript/infra/hardhat.config.ts

```diff
@@ -10,12 +10,12 @@ import { AbacusCoreDeployer } from './src/core';
 import { AbacusContractVerifier } from './src/verify';
 import { sleep } from './src/utils/utils';
 import {
+  getCoreEnvironmentConfig,
   getCoreVerificationDirectory,
   getCoreContractsSdkFilepath,
   getCoreRustDirectory,
-  registerMultiProvider,
-  getCoreConfig,
 } from './scripts/utils';
+import { utils as deployUtils } from '@abacus-network/deploy';
```

Comment:
I think we should push ourselves to better naming so as to avoid this
`utils` is an easy cop out

Reply:
Yeah we have like 4 or 5 utils